lib/resources/r4/medication_request.yaml

```diff
@@ -2,10 +2,28 @@
 name: MedicationRequest
 field_name_base_url: http://hl7.org/fhir/R4/medicationrequest-definitions.html#MedicationRequest
 fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "MedicationRequest"
+    }
+  note: resourceType must be MedicationRequest
+  url: https://hl7.org/fhir/r4/json.html#resources
 - name: status
   required: 'Yes'
   type: code
   description: The current state of the medication request.
+  action: create
+  example: |
+    {
+      "status": "active"
+    }
+  note: Currently we only the support the `active` status.
```

Comment:
rewrite all 'currently we only support the x field' as 

```
When creating a new MedicationRequest, the status must be `active`
```

Reply:
Reworded.

Reply:
status->intent in the second one

Reply:
Fixed: https://github.com/cerner/fhir.cerner.com/pull/353/commits/5b4b8c951640267e7d7ac2be448cdeae49b511d0